crates/re_viewer/src/viewer_analytics.rs

```diff
@@ -23,11 +25,8 @@ pub struct ViewerAnalytics {
 #[cfg(feature = "analytics")]
 impl ViewerAnalytics {
     #[allow(unused_mut, clippy::let_and_return)]
-    pub fn new(
-        disabled: bool,
-        #[cfg(target_arch = "wasm32")] analytics_url: Option<String>,
-    ) -> Self {
-        if disabled {
+    pub fn new(startup_options: &StartupOptions) -> Self {
+        if startup_options.is_in_notebook {
             return Self { analytics: None };
```

Comment:
We should explain here why we return early in notebooks

Reply:
Added comments along the lines of "We only want to have this on `*.rerun.io` domains" here and also in the hostname check.